### PR TITLE
Feature/6697 feedback buttons

### PR DESCRIPTION
**Pull request by amoedoamorim**

### CODE REVIEW

File: src/app/components/UserFeedback.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import Button from 'material-ui-next/Button';
+import Card, { CardContent } from 'material-ui-next/Card';
+import IconButton from 'material-ui-next/IconButton';
+import Tooltip from 'material-ui-next/Tooltip';
+// import Dismiss from 'material-ui/svg-icons/content/clear';
+import FaceFrown from '../../assets/images/feedback/face-frown';
+import FaceNeutral from '../../assets/images/feedback/face-neutral';
+import FaceSmile from '../../assets/images/feedback/face-smile';
```
Comment (amoedoamorim) [inline src/app/components/UserFeedback.js]: Cool way to import and use SVG in the app. Not sure about the path they should live in.

File: src/assets/images/feedback/face-frown.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      display: 'inline-block',
+      height: '48px',
+      width: '48px',
+    }}
+  >
+    <defs>
+      <path id="path-1" d="M0,0h2v4h-2Z" />
+    </defs>
+    <path fill="none" d="M0,0h24v24h-24Z" />
+    <g fill={props.color}>
```
Comment (amoedoamorim) [inline src/assets/images/feedback/face-frown.js]: Cool way to manipulate SVG color :smile:

File: src/app/components/UserFeedback.js (hunk 1 above)
Comment (infojunkie) [reply src/app/components/UserFeedback.js]: that path sounds good

File: src/assets/images/feedback/face-frown.js (hunk 2 above)
Comment (chrisblow) [reply src/assets/images/feedback/face-frown.js]: Awesome!

**Pull request merged**